dev-support/bin/create-release

```diff
@@ -641,7 +641,7 @@ function signartifacts
 
   for i in ${ARTIFACTS_DIR}/*; do
     ${GPG} --use-agent --armor --output "${i}.asc" --detach-sig "${i}"
-    ${GPG} --print-mds "${i}" > "${i}.mds"
+    shasum -a 512 "${i}" > "${i}.sha512"
```

Comment:
If you add `--tag` to this command, the file format is a bit more self-descriptive. The `--tag` option uses BSD-style checksum, rather than GNU coreutils style, which includes the algorithm name in the file contents explicitly. It also avoids the confusing two space/space-star (text vs. binary comparisons) delimiter of GNU style by always doing binary comparisons.

```suggestion
    shasum -a 512 --tag "${i}" > "${i}.sha512"
```

Reply:
Thank you for your review! Reflected your comment.
I used `sha512sum` instead of `shasum` because `shasum` does not have `--tag` option in the release environment.

Reply:
I'm curious, what is your release environment? The author of `shasum` did add the `--tag` option at my suggestion several years ago, so the version in your release environment must be fairly old.

Reply:
shasum `--tag` option is available since perl 6.00, however, the release environment is Ubuntu 16.04 and perl 5.22 is installed.
The Dockerfile used for release/testing: https://github.com/apache/hadoop/blob/trunk/dev-support/docker/Dockerfile
Revision history for Perl extension Digest::SHA: https://metacpan.org/changes/distribution/Digest-SHA